src/dndbeyond/content-scripts/character.js

```diff
@@ -1263,11 +1264,27 @@ async function rollAction(paneClass, force_to_hit_only = false, force_damages_on
             damages[0] = applyGWFIfRequired(action_name, properties, damages[0]);
         }
 
-        const isMeleeAttack = action_name.includes("Polearm Master") || action_name.includes("Pole Strike") || action_name.includes("Unarmed Strike") || action_name.includes("Tavern Brawler Strike")
-        || action_name.includes("Psychic Blade") || action_name.includes("Bite") || action_name.includes("Claws") || action_name.includes("Tail")
-        || action_name.includes("Ram") || action_name.includes("Horns") || action_name.includes("Hooves") || action_name.includes("Talons") 
-        || action_name.includes("Thunder Gauntlets") || action_name.includes("Unarmed Fighting") || action_name.includes("Arms of the Astral Self")
-        || action_name.includes("Shadow Blade") || action_name.includes("Predatory Strike");
+        const isMeleeAttack = [
+            "polearm master",
+            "pole strike",
+            "unarmed strike",
+            "tavern brawler strike",
+            "psychic blade",
+            "bite",
+            "claws",
+            "tail",
+            "ram",
+            "horns",
+            "hooves",
+            "talons",
+            "thunder gauntlets",
+            "unarmed fighting",
+            "arms of the astral self",
+            "shadow blade",
+            "predatory strike",
+            "enhanced unarmed strike",
+            "flurry of blows"
+        ].includes(action_name.toLocaleLowerCase());
```

Comment:
Not quite, because you changed the previous `action_name.includes(x)` into `action_name == x`. I'm not sure, but I **think** som eof these actually match partially the action name, so a safer solution would be:
```js
const isMeleeAttack = [ .... ].some(action => action_name.includes(action))
```

Reply:
yep i can fix that